### PR TITLE
[FIX] stock_account: don't skip the passed value when inheriting

### PR DESCRIPTION
**Pull request by royle-vietnam**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by royleviindoo** [comment]
Hi @seb-odoo , can you review this PR? thanks

**Comment by seb-odoo** [comment]
Hello, I have no idea what this is about, it looks like for either logistics or accounting teams.

File: addons/stock_account/models/account_chart_template.py [hunk 1]
```diff
@@ -13,6 +13,8 @@ class AccountChartTemplate(models.Model):
     @api.model
     def generate_journals(self, acc_template_ref, company, journals_dict=None):
         journal_to_add = [{'name': _('Inventory Valuation'), 'type': 'general', 'code': 'STJ', 'favorite': False, 'sequence': 8}]
+        if journals_dict:
+            journal_to_add.extend(journals_dict)
```
Comment (lordkrandel) [inline addons/stock_account/models/account_chart_template.py:17]:
```suggestion
        journals_dict = (journals_dict or []) + [{'name': _('Inventory Valuation'), 'type': 'general', 'code': 'STJ', 'favorite': False, 'sequence': 8}]
```

**Review by lordkrandel — CHANGES_REQUESTED** [review]
Sorry, this PR got lost in the records of time.
If you could simplify this line here, then it LGTM.

**Comment by royle-viindoo** [comment]
> Sorry, this PR got lost in the records of time. If you could simplify this line here, then it LGTM.

Hi @lordkrandel , I have edited according to your suggestion, please review again

**Comment by lordkrandel** [comment]
I see the commit message and PR description are empty, could you please add:

- Description of the issue/feature this PR addresses 
  -- *What were you doing in Odoo when you noticed this bug?*
- Current behavior before PR 
  -- *a description of what happened with the bug*
- an eventual link to a ticket, if you have opened one

This gives us more context, so that our support could sort out similar tickets and see if other people have had the same problem.

**Comment by lordkrandel** [comment]
Created new PR here: odoo/odoo#181606

**Pull request closed without merging**